### PR TITLE
Utility script

### PR DESCRIPTION
**Pull request by wonderxboy**
A script to run all unit tests at one go
## Description
A script to run all unit tests recursively with "go test -v" mode, exclude the vendor folder.

## Motivation and Context
A one liner shortcut to run all tests

## How Has This Been Tested?
Run several times on my own env - MacOS+gvm(go1.8.3). No failures

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
I think we'd rather have this in the Makefile in the gateway and watchdog folders. Copy the example of the watchdog/Makefile.

**Comment by wonderxboy** [comment]
Not sure what you mean, 1 or 2 or other?
1. Move this script to  watchdog/Makefile ?
2. Use test running scripts from watchdog/Makefile (I don't see scripts running test in this file though)?

File: runtests.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+go test -v $(go list ./... | grep -v /vendor/)
```
Comment (johnmccabe) [inline runtests.sh]: This is no longer needed in 1.9 though right?

https://golang.org/doc/go1.9#vendor-dotdotdot

Comment (wonderxboy) [reply runtests.sh]: yes! Finally ("By popular request..." 😄  ). However, I don't think 1.9 is mainstream yet. It's only been out for 2 months.

**Comment by wonderxboy** [comment]
@alexellis BTW, thanks for helping me changing the title!

File: runtests.sh (hunk 1 above)
Comment (alexellis) [reply runtests.sh]: Bad link. I meant this one https://github.com/openfaas/faas-cli/blob/master/Makefile

Comment (wonderxboy) [reply runtests.sh]: done

**Comment by alexellis** [comment]
Happy for you to open a PR to create a simplistic Makefile for the watchdog. I'll close for now as I am not sure we need another utility script.

**Pull request closed without merging**